Test/Framework/Graphics/SpriteBatchTest.cs

```diff
@@ -296,7 +294,7 @@ public void DrawWithTexture()
         [Test]
         public void DrawWithCustomEffectAndTwoTextures()
         {
-            var customSpriteEffect = AssetTestUtility.CompileEffect(gd, "CustomSpriteBatchEffect.fx");
+            var customSpriteEffect = content.Load<Effect>(Paths.Effect("CustomSpriteBatchEffect"));
```

Comment:
@KonajuGames - How do you feel about this change?

I would say the alternative here is to only pre-compile the shaders for OpenGL and let the tests still compile the shaders at test time for platforms where we can.  This give us a little more test coverage.

That said we do need a good way to support platforms where the effects (and other content) have to be compiled offline.  This is required if we ever want to run some of these tests on mobile devices or consoles.  So there are clearly parts of this we want to keep.